velox/vector/tests/VectorTest.cpp

```diff
@@ -3101,5 +3101,143 @@ TEST_F(VectorTest, appendNulls) {
   // Append negative.
   EXPECT_ANY_THROW(rowVector->appendNulls(-1));
 }
+
+TEST_F(VectorTest, primitiveTypeNullEqual) {
+  auto base = makeNullableFlatVector<int32_t>({1, 2, std::nullopt});
```

Comment:
nit: perhaps, move base and other after lambdas to make it easier to see what values are referred to by indices passed to lambdas.

Reply:
The lambdas could not capture if moved the `base` and `other` after themselves.

Reply:
Got it. Thanks.